Cargo.toml

```diff
@@ -24,6 +24,11 @@ salsa-macros = { path = "components/salsa-macros" }
 smallvec = "1"
 rayon = "1.10.0"
 
+[features]
+# FIXME: remove this as a default feature before 1.0.
+default = ["salsa_unstable"]
```

Comment:
This can wait until a future PR, but I think we want to tweak the testing setup so that this feature is not included for (most) tests.